app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,221 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+private val MovieDetailsMapper: MovieDetailsMapper = MovieDetailsMapperImpl()
+
+val movieDetailsViewState = MovieDetailsMapper.toMovieDetailsViewState(MoviesMock.getMovieDetails())
+
+@Composable
+fun MovieDetailsRoute(
+
+) {
+    val detailsViewState by remember { mutableStateOf(movieDetailsViewState) }
+
+    MovieDetailsScreen(
+        detailsViewState,
+    )
+}
+
+@Composable
+fun MovieDetailsScreen(
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    LazyColumn(
+        modifier = Modifier
+            .background(color = MaterialTheme.colors.background)
+    ) {
+        item {
+            MovieImage(movieDetailsViewState)
+        }
+
+        item {
+            MovieOverview(movieDetailsViewState)
+        }
+
+        item {
+            MovieCrewman(movieDetailsViewState)
+        }
+
+        item {
+            MovieCast(movieDetailsViewState)
+        }
+    }
+}
+
+@Composable
+fun MovieCast(
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    Column {
+        Text(
+            text = "Top Billed Cast",
+            fontSize = 17.sp,
+            fontWeight = FontWeight.Bold,
+            color = MaterialTheme.colors.primary,
+            modifier = Modifier
+                .padding(horizontal = MaterialTheme.spacing.medium, vertical = MaterialTheme.spacing.small)
+        )
+        LazyRow(
+            horizontalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium),
+            modifier = Modifier
+                .padding(horizontal = MaterialTheme.spacing.medium, vertical = MaterialTheme.spacing.small)
+        ) {
+            items(movieDetailsViewState.cast.size) { actor ->
+                ActorCard(
+                    actorCardViewState = ActorCardViewState(
+                        id = movieDetailsViewState.cast[actor].id,
+                        imageUrl = movieDetailsViewState.cast[actor].imageUrl,
+                        name = movieDetailsViewState.cast[actor].name,
+                        character = movieDetailsViewState.cast[actor].character
+                    ),
+                    modifier = Modifier
+                        .size(
+                            width = dimensionResource(id = R.dimen.actor_card_width),
+                            height = dimensionResource(
+                                id = R.dimen.actor_card_height
```

Comment:
Format. This should be in single line, it look very inconsistent when you have different formatting in 2 lines in a row.